src/language-js/printer-estree.js

```diff
@@ -5233,6 +5245,15 @@ function templateLiteralHasNewLines(template) {
   return template.quasis.some(quasi => quasi.value.raw.includes("\n"));
 }
 
+function isStyledComponentsCssTemplate(n, text, options) {
+  return (
+    n.type === "LogicalExpression" &&
+    n.right.type === "TaggedTemplateExpression" &&
+    n.right.tag.name === "css" &&
```

Comment:
This is confusing.. the function is called "is styled components css template" but it checks if it's a `LogicalExpression`.

I say you either rename the function or "expect" the node to be `TaggedTemplateExpression`. And `n.type === "LogicalExpression"` would be moved out of this function, i.e.:

```js
// instead of
isStyledComponentsCssTemplate(n)

// would be
n.type === "LogicalExpression" && isStyledComponentsCssTemplate(n.right)
```

Also, it doesn't seem too restrictive to be called "is styled components css template", looks like "is css template" would be equally descriptive.

Reply:
You’re right. (:

This part will probably be gone if I implement formatting I have shown above.